addons/zylann.hterrain/tools/inspector/inspector.gd

```diff
@@ -207,16 +207,21 @@ func _make_editor(key: String, prop: Dictionary) -> HT_InspectorEditor:
 				var option_button := OptionButton.new()
 				
 				for i in len(prop.enum_items):
-					var item = prop.enum_items[i]
-					option_button.add_item(item)
-				
-				# TODO We assume index, actually
+					var item:Array = prop.enum_items[i]
+					var value:int = item[0]
+					var text:String = item[1]
+					option_button.add_item(text, value)
+
 				getter = option_button.get_selected_id
-				setter = option_button.select
+				setter = func select_id(id: int):
+					var index:int = option_button.get_item_index(id)
+					if index >= 0:
```

Comment:
Can an error be logged if this check doesn't pass? That doesnt seem to be something to expect.
Could either be done with `assert`, or using logger (direct prints aren't used in the plugin)

Reply:
Hm… an `assert` sounds appropriate. Will do